src/main/java/reposense/frontend/CliArguments.java

```diff
@@ -121,6 +134,21 @@ private void setUserInputValuesToArgument(final HashMap<String, String> argument
         }
     }
 
+    /**
+     * Verifies the correctness of input values
+     *
+     * @throws IllegalArgumentException If user-supplied values are invalid
+     */
+    private void verifyInputValues() {
```

Comment:
verifyDateArgs would be better. Also since this method would be used in constructor, I would prefer that you supply the variables in parameters.